doc/airline.txt

```diff
@@ -1662,7 +1674,8 @@ And the following are defined for their respective extensions:
 
 `ale_error_count` `ale_warning_count` `branch` `eclim` `hunks`
 `languageclient_error_count` `languageclient_warning_count` `lsp_error_count`
-`lsp_warning_count` `neomake_error_count` `neomake_warning_count` `obsession`
+`lsp_warning_count` `nvimlsp_error_count` `nvimlsp_warning_count`
+`neomake_error_count` `neomake_warning_count` `obsession`
```

Comment:
Alphabetical order, please, too.

Reply:
Put that in alphabetical order, as well as in other similar comment.